k8s/client.go

```diff
@@ -18,6 +19,8 @@ var (
 	automountServiceAccountToken       = true
 )
 
+const calyptiaFluentBitImage = "ghcr.io/calyptia/core/calyptia-fluent-bit"
```

Comment:
I don't think we want references to this image, this is internal only. use !coreDockerImage